packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -162,6 +162,7 @@ export default defineComponent({
     totalProgress: 0, // current uploads progress (0-100)
     uploadsPaused: false, // all uploads paused?
     uploadsCancelled: false, // all uploads cancelled?
+    uploadsWritingToStorage: false, // uploads transferred but still need to be written to storage
```

Comment:
I'd prefer a more generic variable name that doesn't imply backend knowledge. What do you think about `inFinalization`?

```suggestion
    inFinalization: false, // uploads transferred but still need to be written to storage
```